Automattic/jetpack
# Get rid of `jetpack/publicize` store

Following #34064, this PR cleans up the jetpack social store usage

## Proposed changes:

* Remove `jetpack/publicize` store
* Convert the store sections to re-usable utility functions
* Thus get rid of `lodash` and `refx` deps from `publicize-components` 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to post editor
* Confirm that all the Jetpack Social and SIG related settings work correctly
